archeologies-et-sciences-de-lantiquite.csl

```diff
@@ -0,0 +1,700 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="fr-FR">
+  <info>
+    <title>Archéologies et Sciences de l’Antiquité</title>
+    <title-short>ArScAn</title-short>
+    <id>http://www.zotero.org/styles/archeologies-et-sciences-de-lantiquite</id>
+    <link href="http://www.zotero.org/styles/archeologies-et-sciences-de-lantiquite" rel="self"/>
+    <author>
+      <name>AVSz</name>
+      <email>avszaba@gmail.com</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <summary>Le style ArScAn sert à la revue 'Les Cahiers d'ArScAn' et autres publications d'ArScAn (Archéologies et Sciences de l'Antiquité, CNRS), aux documents d'archéologie et d'histoire de l'art. Il est fait d'emprunts aux styles français du forum francophone de Zotero (invisu, etc.).</summary>
```

Comment:
@avsz, is there a style in particular that you based this one on? If so, please refer to it via a "template" link. See https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link